src/quacc/recipes/emt/elastic.py

```diff
@@ -66,11 +69,21 @@ def bulk_to_deformations_flow(
         [relax_job, static_job],
         param_swaps=job_params,
         decorators=job_decorators,
-    )
+    )  # type: ignore
+
+    if pre_relax:
+        if run_static:
+            undeformed_relax = relax_job_(atoms, relax_cell=True)
+            undeformed_result = static_job_(undeformed_relax["atoms"])
+        else:
+            undeformed_result = relax_job_(atoms, relax_cell=True)
+    else:
+        undeformed_result = static_job_(atoms)
```

Comment:
This code block has some problems. For instance, it will run a static job even if `run_static=False` if `pre_relax=False` too. Also, we have some code duplication here with the `static_job_` call.

This code block could be greatly cleaned up by doing something like the following (note: please sanity check): 

```python
if pre_relax:
    undeformed_result = relax_job_(atoms, relax_cell=True)
    atoms = underformed_result["atoms"]
if run_static:
    underformed_result = static_job_(atoms)
```